docs/3-Contributing/1-Building-Fleet.md

```diff
@@ -41,7 +41,7 @@ sudo npm install -g yarn
 
 To install dependecies, we recommend using [Chocolatey](https://chocolatey.org/install). Chocolatey must be run in Powershell as an Administrator. Assuming your setup does not include any of our requirements, please run:
 ```
-choco install nodejs git golang docker make
+choco install nodejs git golang docker make python2
```

Comment:
Huh, do you remember what failed that made you add this to the list of deps?

Reply:
Heh, I had the same question when we discussed this earlier on the call. Ben said something in our JS stack seems to be using python.

Reply:
```
[4/4] Building fresh packages...
error C:\Users\Ben\source\fleetdm\fleet\node_modules\node-sass: Command failed.
Exit code: 1
Command: node scripts/build.js
Arguments:
Directory: C:\Users\Ben\source\fleetdm\fleet\node_modules\node-sass
Output:
Building: C:\Program Files\nodejs\node.exe C:\Users\Ben\source\fleetdm\fleet\node_modules\node-gyp\bin\node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
gyp info it worked if it ends with ok
gyp verb cli [
gyp verb cli   'C:\\Program Files\\nodejs\\node.exe',
gyp verb cli   'C:\\Users\\Ben\\source\\fleetdm\\fleet\\node_modules\\node-gyp\\bin\\node-gyp.js',
gyp verb cli   'rebuild',
gyp verb cli   '--verbose',
gyp verb cli   '--libsass_ext=',
gyp verb cli   '--libsass_cflags=',
gyp verb cli   '--libsass_ldflags=',
gyp verb cli   '--libsass_library='
gyp verb cli ]
gyp info using node-gyp@3.8.0
gyp info using node@16.5.0 | win32 | x64
gyp verb command rebuild []
gyp verb command clean []
gyp verb clean removing "build" directory
gyp verb command configure []
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` failed Error: not found: python2
gyp verb `which` failed     at getNotFoundError (C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:13:12)
gyp verb `which` failed     at F (C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:68:19)
gyp verb `which` failed     at E (C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:80:29)
gyp verb `which` failed     at C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:89:16
gyp verb `which` failed     at C:\Users\Ben\source\fleetdm\fleet\node_modules\isexe\index.js:42:5
gyp verb `which` failed     at C:\Users\Ben\source\fleetdm\fleet\node_modules\isexe\windows.js:36:5
gyp verb `which` failed     at FSReqCallback.oncomplete (node:fs:195:21)
gyp verb `which` failed  python2 Error: not found: python2
gyp verb `which` failed     at getNotFoundError (C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:13:12)
gyp verb `which` failed     at F (C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:68:19)
gyp verb `which` failed     at E (C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:80:29)
gyp verb `which` failed     at C:\Users\Ben\source\fleetdm\fleet\node_modules\which\which.js:89:16
gyp verb `which` failed     at C:\Users\Ben\source\fleetdm\fleet\node_modules\isexe\index.js:42:5
gyp verb `which` failed     at C:\Users\Ben\source\fleetdm\fleet\node_modules\isexe\windows.js:36:5
gyp verb `which` failed     at FSReqCallback.oncomplete (node:fs:195:21) {
gyp verb `which` failed   code: 'ENOENT'
gyp verb `which` failed }

```